### PR TITLE
Fix zip AppBar

### PR DESCRIPTION
**Pull request by EmmanuelMess**
* Fixed Toolbar title and BottomBar path (#623).
* Fixed root vector on BottomBar to better represent zip relative paths.
* Fixed crash when BottomBar was clicked after opening zip file.
* Cleaned up the code a bit.

### CODE REVIEW

File: app/src/main/java/com/amaze/filemanager/fragments/ZipViewer.java [hunk 1]
```diff
@@ -78,9 +76,19 @@
 import java.util.ArrayList;
 import java.util.Calendar;
 
-public class ZipViewer extends Fragment {
+public class ZipViewer extends Fragment implements BottomBarButtonPath {
```
Comment (szholdiyarov) [inline app/src/main/java/com/amaze/filemanager/fragments/ZipViewer.java]: shouldn't it be "ZipViewFragment"?

Comment (EmmanuelMess) [inline app/src/main/java/com/amaze/filemanager/fragments/ZipViewer.java]: Yes, but I would have to change a lot of fragments to have their correct name and that would make a lot more changes and the pull more prone to conflict. That should be corrected in another pull request (with just naming corrections).

**Comment by vishal0071** [comment]
A little bug that I found,. pressing on bottom bar does show file hierarchy but pressing on any one of them doesn't do anything, also pressing on root icon (zip) force closes Amaze. Can't you look into it?

**Comment by EmmanuelMess** [comment]
@vishal0071 done

**Comment by EmmanuelMess** [comment]
Can anyone check with a rar file?

**Comment by vishal0071** [comment]
Looks good.
